pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/Tissue.java

```diff
@@ -316,8 +316,8 @@ public void setSmIdBasedOnType(SmId tissueSmId, ResultSet rs) {
                 default:
                     logger.error("Unknown type");
             }
-        } catch (SQLException throwables) {
-            throwables.printStackTrace();
+        } catch (SQLException e) {
+            throw new DsmInternalError(String.format("Error setting type for smId with id: ", tissueSmId.getSmIdPk()), e);
```

Comment:
Looks like you are missing `%` in `StringFormat`? (The linter should spot this for you.)
(You can also just skip `StringFormat` here and append in-line.)